db/system_keyspace.cc

```diff
@@ -1499,6 +1499,17 @@ future<std::unordered_map<gms::inet_address, utils::UUID>> system_keyspace::load
     });
 }
 
+future<std::vector<gms::inet_address>> system_keyspace::load_peers() {
```

Comment:
As usual, please provide a rationale in the commit comment - how it's going to be used.